editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -130,6 +139,53 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 	}
 
 	/**
+	 * responsible to update the manifest.mf to use the project name as Bundle-SymbolicName
+	 * 
+	 * @param project
+	 * @param monitor
+	 * @throws CoreException
+	 */
+	protected void updateBundleSymbolicName(IProject project, IProgressMonitor monitor) throws CoreException {
+		List<IFile> files = new ArrayList<>();
+		project.accept(new IResourceVisitor(){
+			@Override
+			public boolean visit(IResource resource) throws CoreException {
+				if( resource instanceof IFile && "manifest.mf".equalsIgnoreCase(resource.getName())) {
+					files.add((IFile)resource);
+				}
+				return true;
+			}
+			
+		});
+		if (!files.isEmpty()) {
+			for (IFile f : files) {
+				try {
+					InputStream is = f.getContents();
+					Manifest mf = new Manifest(is);
+					is.close();
+					Attributes attributes = mf.getMainAttributes();
+					attributes.putValue("Bundle-SymbolicName", getBundleSymbolicNameForProjectName(project.getName()));
+					// lets also update the bundle name so Fuse Runtime shows a difference too
+					attributes.putValue("Bundle-Name", String.format("%s [%s]", attributes.getValue("Bundle-Name"), project.getName()));
+					mf.write(new FileOutputStream(f.getLocation().toFile()));
+				} catch(IOException ioe) {
+					ProjectTemplatesActivator.pluginLog().logError(ioe);
+				}
+			}
+		}
+	}
+	
+	/**
+	 * converts a project name into a bundle symbolic name
+	 * 
+	 * @param projectName
+	 * @return
+	 */
+	protected String getBundleSymbolicNameForProjectName(String projectName) {
+		return projectName.replaceAll(" ", "-");

```

Comment:
are we sure that only space is a non-allowed character which is allowed for project character?